### DISCUSSION TITLE
V8ScriptEngine Process

### DISCUSSION THREAD

**Discussion by AndrewJamesFry** [ClearFoundry/ClearScript#360] [Q&A] [answered]
Hi, does using the V8ScriptEngine spawn a new process? 

We assume it does as it's used in a C# using statement?

Thanks

**Comment by ClearScriptLib** [comment, marked as answer]
Hi @AndrewJamesFry,

>does using the V8ScriptEngine spawn a new process?

No. The V8 implementation and related ClearScript hosting code is native – that is, unmanaged – but it runs in-process with the managed host.

Cheers!
